packages/runner/lib/state.ts

```diff
@@ -1,2 +1 @@
-import type { ChildProcess } from 'child_process';
-export const childProcesses = new Map<string, ChildProcess>();
+export const runningSyncsWithAborts = new Map<string, AbortController>();
```

Comment:
what about `syncAbortControllers` to describe what the map contains?

Reply:
Thanks, updated with https://github.com/NangoHQ/nango/pull/1684/commits/56c3414df31b0d1d85396647e27f757dd9ef936e